server/views/components/feedback-banner.njk

```diff
@@ -0,0 +1,11 @@
+{% macro feedbackBanner(params) %}
+<div class="govuk-!-display-none-print {{ params.feedbackBannerAdditionalClasses }}">
+  <div class="govuk-width-container">
+    <div class="feedback-banner" data-test="feedback-banner" data-qa="feedback-link">
+      <a href="/feedback" class="govuk-link govuk-link--inverse" rel="noopener noreferrer">
+        Suggest an improvement or report a problem with this service
+      </a>
+    </div>
+  </div>
+</div>
+{% endmacro %}
```

Comment:
Think we should add an empty lint at the end of this file to avoid the Nunjucks format error on line 11